tests/unit/modules/dcnm/fixtures/dcnm_intf_multi_intf_configs.json

```diff
@@ -130,7 +130,7 @@
           "no shutdown"
         ],
         "peer1_allowed_vlans": "none",
-	"peer1_native_vlan": "10",
+	      "peer1_native_vlan": "10",
```

Comment:
Looks like white space was introduced here and elsewhere.. was that intentional?  Can you restore the original back?